OpenRA.Game/WAngle.cs

```diff
@@ -79,6 +79,27 @@ public static WAngle Lerp(WAngle a, WAngle b, int mul, int div)
 			return new WAngle(aa + (bb - aa) * mul / div);
 		}
 
+		public static WAngle ChangeByStep(WAngle a, WAngle b, WAngle step_speed)
```

Comment:
```suggestion
		public static WAngle ChangeByStep(WAngle a, WAngle b, WAngle stepSpeed)
```

https://github.com/OpenRA/OpenRA/wiki/Coding-Standard

Reply:
Problems are probably resovled (for I don't view the code, but I believe CastleJing has already fix those). Please look at the new version, now with vehicle.